### ISSUE TITLE
Can large data file be loaded by pages?

### ISSUE THREAD

**Issue by lidanger** [julianpaulozzi/LiteDbExplorer#21]
The project is great, thanks for all your work.

Here, I have a problem about large data file.
Size of my files is 100MB-25GB, while the largest collection can not be load into memory totaly to view.So, can you add some paging features like mongodb clients?

@julianpaulozzi

**Comment by julianpaulozzi** [comment]
Yes depending on if use will have problems (or many), see [Important Notes](https://github.com/julianpaulozzi/LiteDbExplorer#important-notes).

Previously I tested skip and take page pagination, believe me I saw no real benefit and/or improvement after having already activated UI virtualization.

Plans for the future is to add the ability to query using shell commands, which could give you the pagination besides other possibilities and a read-only connection mode to reduce memory consumption.
